src/App.tsx

```diff
@@ -17,18 +67,21 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter filter={filter} onFilter={setFilter} />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length ? (
+                <TodoList todos={filtredTodos} />
+              ) : (
+                <Loader />
+              )}
```

Comment:
Would be better to create state for `isLoading` and change it before and after you make API requests
What if you will get an empty response from API? You will see the loader all the time